libs/blocks/global-navigation/global-navigation.css

```
@@ -798,6 +805,22 @@ header.new-nav .feds-nav-wrapper {
   visibility: hidden;
 }
 
+header.new-nav .feds-nav-wrapper:not(:has(.feds-dropdown--active)) {
```

Can we group the overflow-x: clip; for better readability.
Like - 
```suggestion
header.new-nav .feds-nav-wrapper:not(:has(.feds-dropdown--active)),
header.new-nav.global-navigation:has(.feds-dropdown--active) {
  overflow-x: clip;
}
```

---

done